### ISSUE TITLE
cell_measures for tauuo, tauvo, etc

### ISSUE THREAD

**Issue by martinjuckes** [cmip6dr/CMIP6_DataRequest_VariableDefinitions#31]
Most variables have cell_measures set to areacella, areacello or areacellg. Such variables must be archived with values at cell centres. Some variables may be archived at cell boundaries or veritices. Does this apply to tauuo, tauvo, tauucorr and tauvcorr in the Omon table? See https://www.earthsystemcog.org/projects/wip/drq_interp_cell_center for more discussion of this issue and other variables it applies to.

**Comment by StephenGriffies** [comment]
Here is a recommendation that I think will allow ocean modellers to do the natural thing, and to follow CMIP5 practice.

tauuo and tauucorr should have the same cell_measures as uo
tauvo and tauvcorr should have the same cell_measures as vo

In plain English: tauuo and tauucorr are components to the zonal wind stress.  This stress forces the ocean model zonal velocity, uo, at the ocean surface k=1.  So tauuo and tauucorr live on the same grid as uo at the ocean surface.  Likewise, the same holds for  tauvo and tauvcorr, which live on the same k=1 grid as vo.  So any specifications for uo and vo should be consistent with the corresponding surface wind stresses.   

The above recommendation is what a modeller would naturally wish to provide, as it requires no regridding.

**Comment by taylor13** [comment]
I agree with Steve on this.  This is also consistent with how we treated surface wind stress in CMIP5.

**Comment by martinjuckes** [comment]
Set to --OPT
